src/tarantool/bank.clj

```diff
@@ -37,10 +37,13 @@
                              balance INT NOT NULL)")])
            (doseq [a (:accounts test)]
                (info "Populating account")
-               (sql/insert! conn table-name {:id      a
-                                             :balance (if (= a (first (:accounts test)))
-                                                       (:total-amount test)
-                                                       0)}))))
+               ; Distribute initial balances uniformly.
```

Comment:
We can move `let` construction before `doseq`. It's more than enough to calculate `initial-balance` once.

Reply:
Well, I can just add it into `let` above and don't even increase depth level. Done.